modin/pandas/series.py

```diff
@@ -321,7 +321,11 @@ def __repr__(self):
             len_str = "Length: {}, ".format(len(self.index))
         else:
             len_str = ""
-        dtype_str = "dtype: {}".format(temp_str.rsplit("dtype: ", 1)[-1])
+        dtype_str = "dtype: {}".format(
+            str(self.dtype) + ")"
+            if temp_df.empty
+            else temp_str.rsplit("dtype: ", 1)[-1]
```

Comment:
Can't we make `dtype_str` from `temp_str` in empty Series case?

Reply:
No, because `temp_str` has no dtype at all in it in this case.

```
In [9]: repr(pd.Series([1])._build_repr_df(10, 10).iloc[:, 0])                                UserWarning: Distributing <class 'list'> object. This may take some time.  
Out[9]: '0    1\nName: __reduced__, dtype: int64'

In [11]: repr(pd.Series()._build_repr_df(10, 10))                                             UserWarning: Distributing <class 'NoneType'> object. This may take some time.
Out[11]: 'Empty DataFrame\nColumns: []\nIndex: []'
```

Notice the second case there's no dtype string in there, so there's nothing to extract.